src/main/java/com/squareup/javapoet/AnnotationSpec.java

```diff
@@ -183,13 +185,17 @@ public Builder toBuilder() {
   }
 
   @Override public boolean equals(Object o) {
-    return o instanceof AnnotationSpec
-        && ((AnnotationSpec) o).type.equals(type)
-        && ((AnnotationSpec) o).members.equals(members);
+    if (this == o)
+      return true;
+    if (o == null)
+      return false;
+    if (getClass() != o.getClass())
+      return false;
+    return hashCode == o.hashCode();
```

Comment:
[_Sorry, we found no matches for your search term(s) polygenelubricants._](http://dict.leo.org/ende/index_en.html#/search=polygenelubricants&searchLoc=0&resultOrder=basic&multiwordShowSingle=on)